sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx

```diff
@@ -45,11 +50,19 @@ export const SlippageSelector: FC<SlippageSelectorProps> = () => {
 							<SolidTooltipCustomValue
 								value={customSlippage}
 								onChange={(_, value) => {
-									if (value !== '') {
-										const numValue = Number(value);
-										setSlippage(numValue >= 0 ? numValue : DEFAULT_SLIPPAGE);
+									const numValue = Number(value);
+
+									if (value === '') {
+										setCustomSlippage('');
+										setSlippage(DEFAULT_SLIPPAGE);
+									} else if (numValue === 0) {
+										setCustomSlippage(value);
+										setSlippage(MIN_SLIPPAGE);
```

Comment:
note - altho the custom value here is "0", I don't really think the user will ever want "0" slippage, so I set a min slippage of 0.01 (0 is valid, but will most likely result in a failed tx)

Reply:
it might be better tho, to actually disable 0 slippage input